specification/vmware/resource-manager/Microsoft.AVS/preview/2019-08-09-preview/vmware.json

```diff
@@ -0,0 +1,1277 @@
+            "name": "QuotaEnabled",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Resource": {
+      "description": "The core properties of ARM resources",
+      "properties": {
+        "id": {
+          "description": "Resource ID.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "name": {
+          "description": "Resource name.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type.",
+          "readOnly": true,
+          "type": "string"
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "TrackedResource": {
+      "description": "The resource model definition for a ARM tracked top level resource",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Resource location"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "description": "Resource tags"
+        }
+      }
+    },
+    "ApiError": {
+      "properties": {
+        "error": {
+          "description": "An error returned by the API",
+          "$ref": "#/definitions/ApiErrorBase"
+        }
+      },
+      "description": "API error response"
+    },
+    "ApiErrorBase": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "Error code"
+        },
+        "message": {
+          "type": "string",
+          "description": "Error message"
+        }
+      },
+      "description": "Api error."
+    },
+    "OperationList": {
+      "description": "Pageable list of operations",
+      "properties": {
+        "value": {
+          "description": "List of operations",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Operation"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Operation": {
+      "description": "A REST API operation",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the operation being performed on this object",
+          "type": "string",
+          "readOnly": true
+        },
+        "display": {
+          "description": "Contains the localized display information for this operation",
+          "readOnly": true,
+          "properties": {
+            "provider": {
+              "description": "Localized friendly form of the resource provider name",
+              "type": "string",
+              "readOnly": true
```

Comment:
>"readOnly": true [](start = 14, length = 16)

Thank you for annotating read only properties! It will helps the core platform features like what-if or policy.